process/smartContract/process.go

```diff
@@ -1323,6 +1348,10 @@ func (sc *scProcessor) DeploySmartContract(tx data.TransactionHandler, acntSnd s
 	var vmOutput *vmcommon.VMOutput
 	snapshot := sc.accounts.JournalLen()
 
+	defer func() {
```

Comment:
instead of defer call put at the end of the function

Reply:
done.